mmv1/templates/terraform/examples/gkeonprem_vmware_cluster_f5lb.tf.erb

```diff
@@ -21,6 +21,7 @@ resource "google_gkeonprem_vmware_cluster" "<%= ctx[:primary_resource_id] %>" {
         gateway="test-gateway"
       }
     }
+    vcenter_network = "test-vcenter-network"
```

Comment:
Is this field actually set the network or is this just passing the value? as I don't think network `test-vcenter-network` exists in our project

Reply:
Thanks for checking. This is just passing the value, the test would not actually set the network. @shuyama1